### DISCUSSION TITLE
baikal api

### DISCUSSION THREAD

**Discussion by sadden3194** [ckulka/baikal-docker#126] [Q&A]
Hi there!

Is an api available to create / edit / delete contacts / events?
I am searching for a php solution.

**Comment by ckulka** [comment]
Hi @Ghost108, that's a good question for the repository of Baikal [sabre-io/Baikal](https://github.com/sabre-io/Baikal), this one here is just packaging and providing it as a Docker image :)

They would know if there's an API for it (other than the regular WebDAV API).

**Reply by sadden3194** [reply]
webdav api is also an solution. is there a documentation available?

**Reply by ckulka** [reply]
The only thing I could find was https://sabre.io/dav/standards-support/ from their official documentation showing which WebDAV RFCs they implemented. From there on, I'd imagine you can see what they provide, find the ones you're interested in, and then check online examples/docs for those.

Unfortunately I don't know of any OpenAPI spec, examples, Postman Collections etc. for WebDAV though :/
